docs/contracts/function-modifiers.rst

```diff
@@ -111,6 +111,15 @@ whitespace-separated list and are evaluated in the order presented.
 Modifiers cannot implicitly access or change the arguments and return values of functions they modify.
 Their values can only be passed to them explicitly at the point of invocation.
 
+In function modifiers, it is necessary to specify when you want the function to which the modifier is
```

Comment:
I don't think this is the right place to talk about the fundamental feature of modifiers.
The way they are used is explained in the source code comments of the example contract.
We should either not start with an example but with something like this paragraph instead and then provide a slimmed down example or leave it as it is.

Reply:
Hi Chris,

Thanks for reviewing this.

In my previous PR I made my case for why adding this information adds value:

**Why you made your change**

_When I recently read the Solidity docs, I felt that there wasn't enough information on placeholders used within modifiers.

This question has been [asked before](https://ethereum.stackexchange.com/questions/19171/what-does-underscore-do)._

I largely learnt Solidity by going through the docs during my beginner phase and the modifier not being explicitly explained was the biggest hole in the docs IMO (and clearly by the SO thread others feel the same way). I've had a few friends in the space tell me they felt that this was a hole in the docs too; though how big of an issue it is varies person to person.

While I acknowledge it is in the code comments and everyone _should_ be reading that, not everyone does and thus I feel a little added clarity would be beneficial to beginners.

> We should either not start with an example but with something like this paragraph instead and then provide a slimmed down example or leave it as it is.

I do agree with your point here, perhaps it is too long - originally this was formatted for a separate .rst file and admittedly it can be slimmed down a little more. 

Could you please advise which aspects of the placeholder statement you deem to be most important (for example, usage in code, what it does, ability to use it multiple times, etc.) so that I can make appropriate changes? I will commit them and then tag you again in this PR.

Kind Regards,
Arbiter

Reply:
Thanks for your response! Please use your own judgment wrt what you think is most important.

Reply:
Hey @chriseth I have cut it down to one paragraph as per your earlier guidance.

I feel that this is succinct enough while also offering context to confused readers - probably quite a good balance!

Let me know what you think.

Arby